plugins/palo_alto_cortex_xdr/plugin.spec.yaml

```diff
@@ -4,8 +4,9 @@ products: [insightconnect]
 name: palo_alto_cortex_xdr
 title: Palo Alto Cortex XDR
 description: Stop modern attacks with the industry's first extended detection and response platform that spans your endpoints, network and cloud data
-version: 2.3.0
-supported_versions: ["2022-03-28 Palo Alto Cortex XDR API v1"]
+version: 3.0.0
+connection_version: 1
```

Comment:
The connection version should reflect the version when it is added to an existing plugin. In this case, if there is no update to the connection, then set the connection_version to 2.